lib/posexional/row.ex

```diff
@@ -132,7 +133,9 @@ defmodule Posexional.Row do
       |> elem(0)
       |> Enum.filter(fn {k, _} -> not (k in [:empty_field]) end)
 
-    [{name, res}]
+    if not is_nil(struct_module),
+      do: [struct(struct_module, res)],
```

Comment:
Maybe we should keep the same interface to the outside world? So a tuple with `{name, struct}`